### DISCUSSION TITLE
Tracking for multiple servers and organizations

### DISCUSSION THREAD

**Discussion by miroslavskela** [postalserver/postal#1731] [Help with using Postal]
Hello,
I have question about setting tracking domains. Currently I have few different organizations and in all organizations few different mail servers. Is there a possibility to somehow add one click.example.com additional host, or to add new ones but without always changing caddy file and restarting caddy. Also even if I add additional host always manually, is there some limit for additional hosts. 
If I wasn't clear enough, let me know, I will try to explain better. 
Thank you.

**Comment by willpower232** [comment]
As of version 2, all click tracking hosts require extra configuration from Caddy or similar. If fully restarting Caddy is a problem, you should see if you can use the reload functionality to update the config without having to fully stop it.

Version 1 handled the routing and certificates itself but this was removed as it was quite janky to work with.

**Reply by miroslavskela** [reply]
Thank you @willpower232. Is there some option to have one global click domain for all servers and organizations? With this settings all links work whith track.mydomain.com, but custom domains that should be added to mail servers does not work becuse of tls certificates, if they are not added to caddy configuration. Also, maybe some workaround with wildcards in caddy. That maybe could work. But all mail servers can add only registered domain. Idea behind this would be to add *.mydomain.com in caddy for tracking, and to generate one certificate and then to add tracking domains to servers, serverA.mydomain.com, serverB.mydomain.com.

**Reply by willpower232** [reply]
Tracking domains have to be specific at the server level, not the organisation level as that is where all the information for the links and messages themselves are stored.

As you are handling the certificates yourself in Caddy, if you wanted to make use of a wildcard certificate then it would be entirely your choice to do so and Postal would not need to know.
